### ISSUE TITLE
Automatic replacement of <i />-tags

### ISSUE THREAD

**Issue by ewal** [FortAwesome/ember-fontawesome#40]
I'm trying to understand how to get the automatic replacement to work. Could the upgrade guide be complemented with an example perhaps or some more detailed instructions?

**Comment by robmadole** [comment]
Hey there,

We currently have some pre-release candidates out (we are calling them 5.1.0) that address some of the issues that you've been having.

We are doing the work (and documentation) on the `development` branch:

https://github.com/FortAwesome/ember-fontawesome/tree/development

Would you be willing to give this a try? Check out the UPGRADING.md for detailed instructions.

Also remember that since this is a pre-release you need to use NPM tag `@prerelease` like:

```
$ npm i --save @fortawesome/fontawesome-svg-core@prerelease
```

Please let us know how the upgrade goes and if you run into any problems. We are working hard to get the 5.1.0 release stabilized.

**Comment by ewal** [comment]
Hi,

Unfortunately I can't access the development link in your post. I tried to install the package as you suggested though but with no luck.

**Comment by afriqs** [comment]
Hi,
On the MIGRATING.md file there is an entry to [mix `<i>` and `<svg>` modes](https://github.com/FortAwesome/ember-fontawesome/blob/master/UPGRADING.md#mixed-modes-with-automatic-replacement-of-i-tags-to-svg), what we need. Where do we have to paste the provided code ?

**Comment by afriqs** [comment]
@basz No, I did not and had to stick with FontAwesome 4 :pensive:

**Comment by jrjohnson** [comment]
I put some code in https://github.com/FortAwesome/ember-fontawesome/issues/87#issuecomment-453188657 that works in individual components. You could probably add something like it into the application controller to do the same thing globally.  A contribution to add configuration to do this automatically would be most welcome.

**Comment by jrjohnson** [comment]
I believe this is fixed by #88, but please re-open if I'm wrong.
